### PR TITLE
SALTO-5493: update bundle components table

### PR DESCRIPTION
**Pull request by Shulik95**
_Updated bundle components table to contain more data. Minor fixes to the bundle_components_generator script. Added a script to parse DD csv output to the format we use and a README file added._

---

_None_

---
_Release Notes_: 
_None_

---
_User Notifications_: 
_None_

### CODE REVIEW

File: packages/netsuite-adapter/scripts/bundle_components_generation/README.md
Comment (omrilit) [inline packages/netsuite-adapter/scripts/bundle_components_generation/README.md:1]: Like

File: packages/netsuite-adapter/scripts/bundle_components_generation/bundle_components_generator.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def merge_dictionaries(existing_data, new_bundle_dict):
 def main():
 # Running headless chrome speeds the script, but it's not required. To view the browser, remove the headless option.
   op = webdriver.ChromeOptions()
-  op.add_argument('headless')
+#   op.add_argument('headless')
```
Comment (omrilit) [inline packages/netsuite-adapter/scripts/bundle_components_generation/bundle_components_generator.py:147]: is it on purpose?

**Review by omrilit — COMMENTED** [review]
Looks great!

File: packages/netsuite-adapter/scripts/bundle_components_generation/bundle_components_generator.py (hunk 1 above)
Comment (Shulik95) [reply packages/netsuite-adapter/scripts/bundle_components_generation/bundle_components_generator.py:147]: Nope, fixed it 👌

**Review by omrilit — APPROVED** [review]
LGTM, please make sure to merge once communicated properly

**Pull request merged**